source/contributing/writing-documentation.rst

```diff
@@ -63,6 +63,24 @@ If you want to get the latest translations for your build, run:
 
   mvn clean install -Platest,all
 
+Building the standards docs
+===========================
+
+Documentation about the standards and the editor configuration is built from the GeoNetwork source code using the ``schema-doc`` plugin. So do not edit manually the following files:
+
```

Comment:
.. important::  Documentation about the standards and the editor configuration is built from the GeoNetwork source code using the ``schema-doc`` plugin. So do not manually edit the following files: